fixtures/pullRequestPayload.json

```diff
@@ -1,5 +1,5 @@
 {
-  "event": "pull_request",
+  "name": "pull_request",
```

Comment:
Why this change?

Reply:
This change is made because probot expects to receive a webhook with an object of the structure 
`
{  
name: <event_type>, 
payload: <webhookpayload>
}
`
 https://github.com/oppia/oppiabot/blob/9eaf808aacfe8f8ff97ce46a9b7a0182a3616ae8/spec/oppiabotSpec.js#L110

Reply:
Ok, sg.